docs/dev/placement-api.md

```diff
@@ -0,0 +1,69 @@
+# Node Placement configuration for Template Validator and Node Labeller
+
+1. The cluster admin can modify the Template Validator/Node Labeller CR to inform what nodes the respective operator pods should be deployed on.
+2. Template Validator pods are placed on nodes that expect infra components according to the HCO API. 
```

Comment:
Please remove HCO references, as this API exists regardless of HCO, we need to document our own standalone API.

Reply:
Done